### PR TITLE
Cart Animation

### PR DESCRIPTION
**Pull request by 3Wali**
## Description:

1) Added animation on cart

## ScreenShot

https://github.com/haroonmaqsoodch/ShopGo/assets/139584527/48ac9239-fa4c-4ca5-8be0-4307b474dfb9

### CODE REVIEW

File: ShopGo/cartAnimationViewController.swift [hunk 1]
```diff
@@ -0,0 +1,47 @@
+//
+//  cartAnimationViewController.swift
+//  ShopGo
+//
+//  Created by Abdul wali Latif on 7/22/23.
+//
+
+import UIKit
+
+class cartAnimationViewController: UIViewController {
```
Comment (haroonmaqsoodch) [inline ShopGo/cartAnimationViewController.swift]: First letter should be capital

File: ShopGo/cartAnimationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    
+    @IBOutlet weak var cartImage: UIImageView!
+
```
Comment (haroonmaqsoodch) [inline ShopGo/cartAnimationViewController.swift]: remove the spaces between outlets

File: ShopGo/cartAnimationViewController.swift [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+//
+
+import UIKit
+
+class cartAnimationViewController: UIViewController {
+    
+    @IBOutlet weak var cartImage: UIImageView!
+    
+    @IBOutlet weak var shopGoLabel: UILabel!
+
```
Comment (haroonmaqsoodch) [inline ShopGo/cartAnimationViewController.swift]: remove this space

File: ShopGo/cartAnimationViewController.swift [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            UIView.animate(withDuration:0.5){
+                self.shopGoLabel.layer.opacity = 1
+            }
+            
+            
+            
+        }
+        
+        
+    }
```
Comment (haroonmaqsoodch) [inline ShopGo/cartAnimationViewController.swift]: remove the extra spaces

**Review by haroonmaqsoodch — COMMENTED** [review]
Upon running the app, the animation is not showing up and you might not need to add the navigation controller in order to make it work

File: ShopGo.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -24,6 +26,8 @@
 		2A0BD8392A688D4700A8A80E /* Assets.xcassets */ = {isa = PBXFileReference; lastKnownFileType = folder.assetcatalog; path = Assets.xcassets; sourceTree = "<group>"; };
 		2A0BD83C2A688D4700A8A80E /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
 		2A0BD83E2A688D4700A8A80E /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
+		882D12BE2A6C8D3200546A99 /* cart Animation Storyboard.storyboard */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; path = "cart Animation Storyboard.storyboard"; sourceTree = "<group>"; };
+		882D12C02A6C8D7100546A99 /* cartAnimationViewController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = cartAnimationViewController.swift; sourceTree = "<group>"; };
```
Comment (haroonmaqsoodch) [inline ShopGo.xcodeproj/project.pbxproj]: There should not be any spaces in storyboard's name and also put them in a folder

**Review by haroonmaqsoodch — COMMENTED** [review]
You should have your jira story link in the round braces
Screenshot 2023-07-22 at 11 29 32 PM

File: ShopGo/CartAnimation/CartAnimationViewController.swift [hunk 6]
```diff
@@ -0,0 +1,39 @@
+//
+//  cartAnimationViewController.swift
+//  ShopGo
+//
+//  Created by Abdul wali Latif on 7/22/23.
+//
+
+import UIKit
+
+class cartAnimationViewController: UIViewController {
```
Comment (05Waleed) [inline ShopGo/CartAnimation/CartAnimationViewController.swift:10]: First letter should be capital

File: ShopGo/CartAnimation/CartAnimationViewController.swift
Comment (05Waleed) [inline ShopGo/CartAnimation/CartAnimationViewController.swift:1]: There should be only one view controller for cart animation.

File: ShopGo/cart Animation Storyboard.storyboard
Comment (05Waleed) [inline ShopGo/cart Animation Storyboard.storyboard:1]: where is the main storyboard?

**Comment by zuhair5** [comment]
Where is your main storyboard & view controller?

File: ShopGo.xcodeproj/project.pbxproj
Comment (05Waleed) [inline ShopGo.xcodeproj/project.pbxproj:1]: The assets folder is missing, which is causing an issue with displaying the image.

https://github.com/haroonmaqsoodch/ShopGo/assets/139551937/df430875-4033-45d1-8025-70d67823d941

File: ShopGo/cart Animation Storyboard.storyboard
Comment (3Wali) [reply ShopGo/cart Animation Storyboard.storyboard:1]: fixed

File: ShopGo.xcodeproj/project.pbxproj
Comment (3Wali) [reply ShopGo.xcodeproj/project.pbxproj:1]: fixed

**Pull request merged**